test_scripts/ATF_ShutDown_MASTER_RESET.lua

```diff
@@ -0,0 +1,149 @@
+--  Requirement summary:
```

Comment:
@LitvinenkoIra,
To be more clear description and based on tempalte, you can list as exmaple:

-- Requirement summary:
-- [Policies] Master Reset
--.
-- Description:
-- On Master Reset,Policy Manager must revert Local Policy Table to the Preload Policy Table.
--.
-- 1. Used preconditions
-- SDL and HMI are running
-- App1 is registered
--.
-- 2. Performed steps
-- Perform Master Reset
-- HMI sends OnExitAllApplications with reason MASTER_RESET
--.
--  Expected result:
--  1. SDL clear all Apps folder, app_info.dat file and shut down